File: nos2jdbc-tutorial-spring/src/main/java/nos2jdbc/tutorial/spring/entity/nonauto/Ym.java
package nos2jdbc.tutorial.spring.entity.nonauto;

import java.util.List;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.OneToMany;
import nos2jdbc.annotation.NonAuto;

@Entity @NonAuto
public class Ym {
    @Id
    public Integer y;
    @Id
    public Integer m;
    
    @OneToMany(mappedBy = "ym")
    public List<YmItem> itemList;
    

}

File: nos2jdbc-tutorial-kotlin-spring/src/main/kotlin/nos2jdbc/tutorial/kotlinspring/entity/nonauto/Ym.kt
package nos2jdbc.tutorial.kotlinspring.entity.nonauto

import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.OneToMany

import nos2jdbc.annotation.NonAuto

@Entity @NonAuto
class Ym {
    @Id
    var  y: Int? = null
    @Id
    var  m: Int? = null
    
    @OneToMany(mappedBy = "ym")
    var itemList: List<YmItem> = mutableListOf()
    

}
